server/config.js

```diff
@@ -30,11 +23,10 @@ module.exports = {
     gitDate: getEnv('GIT_DATE', '2020-06-21 12:12:12'),
   },
   cookieSecret: getEnv('COOKIE_SECRET', 'keyboard cat'),
+  singleHostName: getRequiredEnv('SINGLE_HOST_NAME', 'localhost'),
   auth: {
-    clientId: getEnv('AZURE_AD_CLIENT_ID', { requireInProduction: true }),
-    clientSecret: getEnv('AZURE_AD_CLIENT_SECRET', {
-      requireInProduction: true,
-    }),
```

Comment:
these were broken as they required a default var so used the ` { requireIn...` as the default value.
Extracted out a `getRequiredEnv ` instead and provided noddy defaults